<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcpserver</class>
 <widget class="QMainWindow" name="tcpserver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1095</width>
    <height>886</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="3">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>接收窗口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEditRecv"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>发送窗口</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPlainTextEdit" name="plainTextEditSend"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonSend">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>发送</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>网络设置</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>本机IP地址</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditIP"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>本地端口号</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditPort"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonDisconnect">
          <property name="text">
           <string>断开</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonClearWindow">
            <property name="text">
             <string>清除窗口</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>FileTransmission</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="pushButton_chooseFile">
             <property name="text">
              <string>选择文件</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_sendFile">
             <property name="text">
              <string>发送文件</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_FileName"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>control panel</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="coverLabel">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="imagefile.qrc">:/image/image/image/non-music.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>285</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="btnPre">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="toolTip">
                <string>上一首</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
background-image: url(:/image/image/image/pre.png);
background-repeat:no-repeat;
background-position:center center;
border:none;
}
QPushButton:hover{
background-image: url(:/image/image/image-hover/pre-hover.png);
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-image: url(:/image/image/image/pre.png);
background-repeat:no-repeat;
background-position:center center;
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="shortcut">
                <string>Alt+Left</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnPlay">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="toolTip">
                <string>播放/暂停</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
background-image: url(:/image/image/image/pase.png);
background-repeat:no-repeat;
background-position:center center;
border:none;
}

QPushButton:hover{
background-image: url(:/image/image/image-hover/pase-hover.png);
background-repeat:no-repeat;
background-position:center center;
}

QPushButton:pressed{
background-image: url(:/image/image/image/pase.png);
background-repeat:no-repeat;
background-position:center center;
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="shortcut">
                <string>Space</string>
               </property>
               <property name="flat">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="btnNext">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>35</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="toolTip">
                <string>下一首</string>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton
{
background-image: url(:/image/image/image/next.png);
background-repeat:no-repeat;
background-position:center center;
border:none;
}
QPushButton:hover{
background-image: url(:/image/image/image-hover/next-hover.png);
background-repeat:no-repeat;
background-position:center center;
}
QPushButton:pressed{
background-image: url(:/image/image/image/next.png);
background-repeat:no-repeat;
background-position:center center;
}
</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="shortcut">
                <string>Alt+Right</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QSlider" name="volumeSlider">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>170</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true"> QSlider 
    {  
        background-color: rgba(255,255,255,0.2);
        min-width:5px; 
        max-width:5px;
        border:15px solid rgba(255,255,255,0.2); 
    }
     QSlider::add-page:vertical
     {      
       
	    background-color: rgb(250, 250, 0);
        width:4px;
     }
     QSlider::sub-page:vertical  
    {
        background-color: rgba(255,255,255,0.5); 
        width:4px;
     }
    QSlider::groove:vertical 
    {
        background:transparent;
        width:6px;
    }
    QSlider::handle:vertical 
    {
         height: 13px;
        width:13px;
 
 margin: -0 -4px; 
	border-image: url(:/image/image/image/volumeHandle.png);
     }</string>
             </property>
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnVolume">
             <property name="minimumSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="toolTip">
              <string>音量控制</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton
{
background-image: url(:/image/image/image/music-voice.png);
background-repeat:no-repeat;
background-position:center center;
border:none;
}

QPushButton:hover{
background-image: url(:/image/image/image-hover/music-voice-hover.png);
background-repeat:no-repeat;
background-position:center center;
}

QPushButton:pressed{
background-image: url(:/image/image/image/music-voice.png);
background-repeat:no-repeat;
background-position:center center;
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1095</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineEditIP</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>pushButtonDisconnect</tabstop>
  <tabstop>pushButtonClearWindow</tabstop>
  <tabstop>plainTextEditSend</tabstop>
  <tabstop>plainTextEditRecv</tabstop>
 </tabstops>
 <resources>
  <include location="imagefile.qrc"/>
 </resources>
 <connections/>
</ui>
